docs/how-to-guides/access-monitoring.md

```diff
@@ -0,0 +1,82 @@
+# Access Monitoring on Lokomotive
+
+## Contents
+
+* [Introduction](#introduction)
+* [Presumptions](#presumptions)
+* [Prometheus](#prometheus)
+* [Alertmanager](#alertmanager)
+* [Grafana](#grafana)
+  * [Using Port Forward](#using-port-forward)
+  * [Using Ingress](#using-ingress)
+
+## Introduction
+
+You can deploy the monitoring stack shipped with lokomotive as shown in the Prometheus Operator component [guide](../configuration-reference/components/prometheus-operator.md).
+
+## Presumptions
+
+- Prometheus Operator component is deployed in `monitoring` namespace. If you have specified a different namespace in config then change the commands accordingly, `kubectl -n <your namespace> ...`.
+
+## Prometheus
+
+Run following command to port forward Prometheus locally on port `9090`.
+
+```
+kubectl -n monitoring port-forward svc/prometheus-operator-prometheus 9090:9090
+```
+
+Now goto [http://localhost:9090](http://localhost:9090).
+
+## Alertmanager
+
+Run following command to port forward Alertmanager locally on port `9093`.
+
+```
+kubectl -n monitoring port-forward svc/prometheus-operator-alertmanager 9093:9093
+```
+
+Now goto [http://localhost:9093](http://localhost:9093).
+
+## Grafana
+
+### Using Port Forward
+
+Run following command to port forward Grafana dashboard locally on port `8080`.
+
+```
+kubectl -n monitoring port-forward svc/prometheus-operator-grafana 8080:80
+```
+
+Obtain Grafana `admin` user password by running following command:
+
+```
+kubectl -n monitoring get secret prometheus-operator-grafana -o jsonpath='{.data.admin-password}' | base64 -d && echo
+```
+
+Now goto [http://localhost:8080](http://localhost:8080). Enter username `admin` and password obtained from previous step.
+
+### Using Ingress
+
+Make sure you are using a configuration for Grafana in Prometheus Operator something similar to following:
```

Comment:
I am not sure about this block. 
What else could be the difference in the configuration ? Its a configuration of our component.

Am I right in saying that you want the the user to use the correct `grafana.ingress.host` value in the component and also when following the docs ? Why not point to the `prometheus-operator` component configuration reference documentation ?

Reply:
Since this is how to guide, we are talking about scenarios. By that logic we can always point folks to the config ref doc to do anything.

The motivation of this doc is to show what various combination of config can allow user to do.